website/docs/guides/core-to-cloud-1.md

```diff
@@ -241,7 +241,7 @@ You have learned:
 - How to configure your development, orchestration, and CI/CD environments
 - How to set up environment variables and validate your models. 
 
-For next steps, we'll soon share other guides on how to manage your move and tips/faqs. Stay tuned!
+Review the next guide, [Move from dbt Core to dbt Cloud: Everything you need to know](/guides/core-to-cloud-2), in this series to learn about the strategic considerations you need to take to move from dbt Core to dbt Cloud.
```

Comment:
This sounds like something they should read prior to this guide. Putting it at the end is a good idea, but we should possibly put it at the start as well. "If you haven't already read..." or something like that.

Reply:
addressed in this pr: https://github.com/dbt-labs/docs.getdbt.com/pull/5158/files